config.md

```diff
@@ -231,4 +231,160 @@ If a hook returns a non-zero exit code, then an error is logged and the remainin
 `args` and `env` are optional.
 The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://golang.org/pkg/os/exec/#Cmd).
 
+## Configuration Schema Example
+
+Here is a full example `config.json` for reference.
+
+```json
+{
+	"ociVersion": "0.3.0",
+	"platform": {
+		"os": "linux",
+		"arch": "amd64"
+	},
+	"process": {
+		"terminal": true,
+		"user": {
+			"uid": 1,
+			"gid": 1,
+			"additionalGids": [5, 6]
+		},
+		"args": [
+			"sh"
+		],
+		"env": [
+			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+			"TERM=xterm"
+		],
+		"cwd": "/",
+		"capabilities": [
+			"CAP_AUDIT_WRITE",
+			"CAP_KILL",
+			"CAP_NET_BIND_SERVICE"
+		],
+		"apparmorProfile": "",
+		"selinuxProcessLabel": ""
```

Comment:
Before these moved [@mrunalp and I were suggesting we drop the empty values][1].  Since #329 [they're omitempty][2], so I see no reason to keep them in this example.

[1]: https://github.com/opencontainers/specs/pull/276#discussion_r52622034
[2]: https://github.com/opencontainers/specs/pull/329/files#diff-b4bda758a2aef091432646c354b4dc59R39